sdk/python/kfp/_client.py

```diff
@@ -82,6 +83,11 @@ class Client(object):
   IN_CLUSTER_DNS_NAME = 'ml-pipeline.{}.svc.cluster.local:8888'
   KUBE_PROXY_PATH = 'api/v1/namespaces/{}/services/ml-pipeline:http/proxy/'
 
+  LOCAL_KFP_CONTEXT = os.path.expanduser('~/.config/kfp/context.json')
+  context_setting = {
```

Comment:
I see writes to this variable, but I see no reads. How is this used?

Reply:
It is read by `get_user_namespace`, see line 241 below.